docs-md/development/jemalloc.md

```diff
@@ -46,6 +60,10 @@ SMR (`mtev_memory_`) and libck's hash tables (`ck_hs_`), one could run:
 
 `jeprof --text --lines --exclude='(mtev_memory_|ck_hs_)' /path/to/your/executable profile.prof`
 
+To compare two heap profiles (which helps reduce the noise) you can use the "--base" switch:
+
+`jeprof --text --lines --exclude=&apos;(mtev_memory_|ck_hs_)&apos; --base=baseline.prof /path/to/your/executable profile.prof`
```

Comment:
Rather than repeating the `--exclude` example above, why not simplify, since this is a different example, unrelated to excluding allocations:

```
jeprof --text --lines --base=profile1.prof /path/to/your/executable profile2.prof
```

Reply:
removed the redundant commandline, pointing out to just add the switch.